src/teststack/commands/containers.py

```diff
@@ -351,6 +351,108 @@ def tag(ctx):
     click.echo(ctx.obj['tag'])
 
 
+def _process_steps(steps):
+    """
+    Process step information from teststack.toml and convert it to a data blob
+    that can be processed in order.
+    """
```

Comment:
Considering converting these into objects that get returned instead of unformatted dictionaries.